cmd/minishift/cmd/start.go

```diff
@@ -371,44 +285,31 @@ func initClusterUpFlags() {
 	clusterUpFlagSet.String(startFlags.NoProxyList.Name, "", "List of hosts or subnets for which no proxy should be used.")
 	clusterUpFlagSet.AddFlag(httpProxyFlag)
 	clusterUpFlagSet.AddFlag(httpsProxyFlag)
+
+	return clusterUpFlagSet
 }
 
 // initProxyFlags create the CLI flags which needs to be passed for proxy
-func initSubscriptionManagerFlags() {
+func initSubscriptionManagerFlags() *flag.FlagSet {
+	var subscriptionManagerFlagSet = flag.NewFlagSet(commandName, flag.ContinueOnError)
```

Comment:
Why not `subscriptionManagerFlagSet := flag.NewFlagSet(commandName, flag.ContinueOnError)`?